app/_src/production/dp-config/cni.md

```diff
@@ -354,6 +348,14 @@ and have `cgroup2` available
 --set "{{site.set_flag_values_prefix}}experimental.ebpf.enabled=true"
 ```
 
-### Merbridge CNI with eBPF Logs
+## {{site.mesh_product_name}} CNI Logs
+
+{% if_version lte:2.1.x %}
+Logs of the CNI plugin are available in `/tmp/kuma-cni.log` on the node and the logs of the installer are available via `kubectl logs`.
+
+If you are using the CNI v2 or eBPF version logs will be available via `kubectl logs` instead.
```

Comment:
I don't think merbridge has logs sorted out. This did not work when I tried it last time.

Reply:
@bartsmykla what's the story here? I simply reordered these lines :)

Reply:
are  you sure? in the original it says:

```
Logs of the installer of Merbridge CNI with eBPF are available via `kubectl logs`.
```
<img width="705" alt="image" src="https://github.com/kumahq/kuma-website/assets/2753650/fa3499d7-c721-4afc-91b5-3579e3f03415">

and you changed it to 
```
If you are using the CNI v2 or eBPF version logs will be available via `kubectl logs` instead.
```
<img width="721" alt="image" src="https://github.com/kumahq/kuma-website/assets/2753650/c266d251-495f-423e-a5e5-9968be379d4c">

and for eBPF (merbridge) only installer logs are available via kubectl.

If I'm missing something obvious let me know.

Reply:
Ha right! You are right!